### PR TITLE
Removed Example app and replaced with quickstart submodule

### PR DESCRIPTION
**Pull request by AshishSynoriq**
If someone wants to have the quick start app, he will need to initialise git submodule.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,3 +8,4 @@ build/
 pubspec.lock
 .gradle/
 local.properties
+example/pubspec.lock
```
Comment (ferologics) [inline .gitignore:11]: this should be removed, no?

**Comment by ferologics** [comment]
We should note that the quickstart app is available as a submodule in the README.

**Comment by ferologics** [comment]
It looks like this PR contains all commits from your master branch, which includes null-safety, etc. Please fix the commit history so that the PR only pertains to the removal of the example.

**Comment by AshishSynoriq** [comment]
Closing this to create a new one

**Pull request closed without merging**